lib/components/user/monitored-trip/trip-notifications-pane.js

```diff
@@ -159,7 +161,7 @@ class TripNotificationsPane extends Component {
                   <option value={45}>45 min</option>
                   <option value={60}>1 hour</option>
                 </Select>
-                before it begins until it ends.
+                before it begins, until it ends.
```

Comment:
Not sure about the inclusion of this comma. I would say we should probably remove it.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/814d9df6102f1c34cf1fedb8c45ba57bb3f24233.